### PR TITLE
[DLS-4468][Suresh]: Play 2.8 upgrade

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

**Comment by sureshhmrc** [comment]
test this please

**Comment by sureshhmrc** [comment]
test this please

File: project/plugins.sbt [hunk 1]
```diff
@@ -3,12 +3,12 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
 resolvers += Resolver.jcenterRepo
 resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("uk.gov.hmrc"               % "sbt-auto-build"     % "3.0.0")
+addSbtPlugin("uk.gov.hmrc"               % "sbt-auto-build"     % "3.3.0")
```
Comment (rambabu-posa) [inline project/plugins.sbt:6]: We can use this version `3.4.0`

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -3,27 +3,29 @@ import sbt._
...
+    "uk.gov.hmrc"                             %% "simple-reactivemongo"      % s"8.0.0-$playVersion",
+    "uk.gov.hmrc"                             %% s"bootstrap-backend-$playVersion" % "5.12.0",
+    "uk.gov.hmrc"                             %% "work-item-repo"            % s"8.1.0-$playVersion",
     "org.typelevel"                           %% "cats-core"                 % "2.6.0",
-    "org.julienrf"                            %% "play-json-derived-codecs"  % "9.0.0",
-    "com.github.kxbmap"                       %% "configs"                   % "0.4.4",
+    "org.julienrf"                            %% "play-json-derived-codecs"  % "10.0.2",
+    "com.github.kxbmap"                       %% "configs"                   % "0.6.1",
     "com.googlecode.owasp-java-html-sanitizer" % "owasp-java-html-sanitizer" % "20191001.1",
     "com.github.ghik"                          % "silencer-lib"              % "1.7.1" % Provided cross CrossVersion.full
```
Comment (rambabu-posa) [inline project/AppDependencies.scala:17]: We can use this version
`"silencer-lib" % "1.7.5"`

File: build.sbt [hunk 3]
```diff
@@ -42,7 +41,7 @@ lazy val microservice = Project(appName, file("."))
     addCompilerPlugin("org.typelevel"  %% "kind-projector"  % "0.13.0" cross CrossVersion.full),
     addCompilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full)
```
Comment (rambabu-posa) [inline build.sbt:42]: We can use this version
`"silencer-lib" % "1.7.5"`

Comment (sureshhmrc) [reply build.sbt]: fixed versions

**Review by rambabu-posa — APPROVED** [review]
LGTM, wait for QA results

**Pull request merged**